components/switch/index.css

```diff
@@ -420,5 +422,6 @@
 		--highcontrast-switch-background-color-selected-disabled: GrayText;
 
 		--highcontrast-switch-focus-indicator-color: ButtonText;
+		forced-color-adjust: none;
```

Comment:
I noticed this had been added to this force-colors query in the switch migration: [S2 switch migration](https://github.com/adobe/spectrum-css/pull/2651)